index.html

```diff
@@ -0,0 +1,801 @@
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Active Volcanoes Around the World That Could Erupt at Any Moment</h3>
+                    </article>
+                    <article class="my-3 col-md-4">
+                        <div class="position-relative">
+                            <img class="article-image"
+                                src="https://d.newsweek.com/en/full/1547486/lipschutz-living-room-hotel-chelsea.webp?w=397&h=265&f=fb4270bf5fbc2591cc43d24d0377621d"
+                                alt="featured-stories">
+                            <img class="hashtag position-absolute" src="./imgs/slideshow-num-icon.png" alt="">
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Inside the Last Bohemian Apartments of the Storied Hotel Chelsea</h3>
+                    </article>
+                    <article class="my-3 col-md-4">
+                        <div class="position-relative">
+                            <img class="article-image"
+                                src="https://d.newsweek.com/en/full/1560097/tozeur-tunisia.webp?w=397&h=265&f=69a77ef1bd7e21c060a7b7685b05a73d"
+                                alt="featured-stories">
+                            <img class="hashtag position-absolute" src="./imgs/slideshow-num-icon.png" alt="">
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Relive the 'Star Wars' Saga at These Filming Locations</h3>
+                    </article>
+                    <article class="my-3 col-md-4">
+                        <div class="position-relative">
+                            <img class="article-image"
+                                src="https://d.newsweek.com/en/full/1552749/eyjafjallajokull-iceland.webp?w=397&h=265&f=ea1ffc67b32c527ef435529cb5bd2dc6"
+                                alt="featured-stories">
+                            <img class="hashtag position-absolute" src="./imgs/slideshow-num-icon.png" alt="">
+                        </div>
+                        <h3 class="font-weight-bold my-1 ">Active Volcanoes Around the World That Could Erupt at Any Moment</h3>
+                    </article>
+                </div>
+            </section>
+            <hr>
+
+            <section class="category py-3">
+                <h3 class="category-title">U.S.</h3>
+                <div class="category-container row">
+                    <article class="col-md-4">
+                        <img class="category-main-img" src="https://d.newsweek.com/en/full/1573074/coronavirus-recoveries-us-global-cases.webp?w=397&h=265&f=0e215f9d6b052a4451c36c89b8126425"
+                            alt="featured-stories">
+                        <h3 class="pt-2 pt-md-0">Over 66,000 Have Recovered From Coronavirus, Per Johns Hopkins University</h3>
+                    </article>
```

Comment:
- [ ] The images on the main articles of these sections (the ones you are not showing yet) should be a little smaller in the medium and small breakpoints.